lib/blogs_api/user.ex

```diff
@@ -0,0 +1,21 @@
+defmodule BlogsApi.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+
+  schema "users" do
+    field :display_name, :string
+    field :email, :string
+    field :password, :string
+    field :image, :string
+    timestamps()
+  end
+
+  @required_params [:display_name, :email, :password, :image]
```

Comment:
Uma dica: nos nossos projetos, costumamos usar sigils para fazer [listas de palavras](https://elixir-lang.org/getting-started/sigils.html#word-lists) nos parâmetros do schema.
```suggestion
  @required_params ~w(display_name email password image)a
```